### PR TITLE
MSSQL: fix tests

### PR DESCRIPTION
**Pull request by alexkart**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  |

### CODE REVIEW

File: tests/framework/db/CommandTest.php [hunk 1]
```diff
@@ -472,7 +472,11 @@ public function testNoTablenameReplacement()
                 'address' => 'Some {{%weird}} address',
             ]
         )->execute();
-        $customerId = $db->getLastInsertID();
+        if ($this->driverName === 'pgsql') {
```
Comment (samdark) [inline tests/framework/db/CommandTest.php]: How did it work before?

Comment (alexkart) [reply tests/framework/db/CommandTest.php]: It inserted id explicitly but MSSQL doesn't allow inserting of identity columns.

Comment (samdark) [reply tests/framework/db/CommandTest.php]: How's that connected to pgsql? It was called without arguments before and tests for PostgreSQL passed.

Comment (alexkart) [reply tests/framework/db/CommandTest.php]: No, previously `'id' => 43,` was inserted and then used in select queries. I've removed `id` from insert statement and added `$db->getLastInsertID()`.

Comment (samdark) [reply tests/framework/db/CommandTest.php]: Ah. OK then.

**Comment by samdark** [comment]
Merged. Thank you very much!

**Pull request merged**